modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -65,11 +65,11 @@ If you do not have an {product-title} cluster, it asks you for the public URL th
 [options="header"]
 |===
 
-| Command | Description 
+| Command | Description
 
-| `tkn pac repo create` | Creates a new {pac} repository and a namespace based on the pipeline run template. 
+| `tkn pac create repository` | Creates a new {pac} repository and a namespace based on the pipeline run template.
 
-| `tkn pac repo list` | Lists all the {pac} repositories and displays the last status of the associated runs.
+| `tkn pac list` | Lists all the {pac} repositories and displays the last status of the associated runs.
```

Comment:
These are the only two lines that need review. 
All the other changes on this page are caused by automated whitespace removal. (The previous contributor didn't have this feature enabled when they created the content.)

Reply:
/unhold